src/Ansatz.jl

```diff
@@ -274,8 +274,11 @@ Truncate the dimension of the virtual `bond` of a [`NonCanonical`](@ref) Tensor
     - `threshold`: The threshold to truncate the bond dimension.
     - `maxdim`: The maximum bond dimension to keep.
     - `compute_local_svd`: Whether to compute the local SVD of the bond. If `true`, it will contract the bond and perform a SVD to get the local singular values. Defaults to `true`.
+    - `renormalize`: Whether to renormalize the state at the bond after truncation. Defaults to `false`.
 """
-function truncate!(::NonCanonical, tn::AbstractAnsatz, bond; threshold, maxdim, compute_local_svd=true)
+function truncate!(
+    ::NonCanonical, tn::AbstractAnsatz, bond; threshold, maxdim, compute_local_svd=true, renormalize=false
```

Comment:
rename `renormalize` to `normalize` to keep consistency with Julia

```suggestion
    ::NonCanonical, tn::AbstractAnsatz, bond; threshold, maxdim, compute_local_svd=true, normalize=false
```